source/backend/tests/unit/api/Services/AcquisitionServiceTest.cs

```diff
@@ -219,14 +223,51 @@ public void Update_Success_UserOverride()
             var repository = _helper.GetService<Mock<IAcquisitionFileRepository>>();
             repository.Setup(x => x.Update(It.IsAny<PimsAcquisitionFile>())).Returns(acqFile);
             repository.Setup(x => x.GetRowVersion(It.IsAny<long>())).Returns(1);
-
+            repository.Setup(x => x.GetById(It.IsAny<long>())).Returns(new PimsAcquisitionFile()
+            {
+                AcquisitionFileStatusTypeCode = acqFile.AcquisitionFileStatusTypeCode,
+            });
             // Act
             var result = service.Update(acqFile, userOverride: true);
 
             // Assert
             repository.Verify(x => x.Update(It.IsAny<PimsAcquisitionFile>()), Times.Once);
         }
 
+        [Fact]
+        public void Update_Success_AddsNote()
+        {
+            // Arrange
+            var service = CreateAcquisitionServiceWithPermissions(Permissions.AcquisitionFileEdit);
+
+            var acqFile = EntityHelper.CreateAcquisitionFile();
+            acqFile.ConcurrencyControlNumber = 1;
+            acqFile.AppCreateUserid = "TESTER";
+
+            var repository = _helper.GetService<Mock<IAcquisitionFileRepository>>();
+            var noteRepository = _helper.GetService<Mock<IEntityNoteRepository>>();
+            var lookupRepository = _helper.GetService<Mock<ILookupRepository>>();
+
+            repository.Setup(x => x.Update(It.IsAny<PimsAcquisitionFile>())).Returns(acqFile);
+            repository.Setup(x => x.GetRowVersion(It.IsAny<long>())).Returns(1);
+            repository.Setup(x => x.GetById(It.IsAny<long>())).Returns(new PimsAcquisitionFile()
+            {
+                AcquisitionFileStatusTypeCode = "CLOSED",
+                AcquisitionFileStatusTypeCodeNavigation = new PimsAcquisitionFileStatusType() { Description = "Closed" }
+            });
+            lookupRepository.Setup(x => x.GetAllAcquisitionFileStatusTypes()).Returns(new PimsAcquisitionFileStatusType[]{ new PimsAcquisitionFileStatusType() {
+                Id = acqFile.AcquisitionFileStatusTypeCodeNavigation.Id,
+                Description = acqFile.AcquisitionFileStatusTypeCodeNavigation.Description,
+            }});
+
+            // Act
+            var result = service.Update(acqFile, true);
+
+            // Assert
+            repository.Verify(x => x.Update(It.IsAny<PimsAcquisitionFile>()), Times.Once);
+            noteRepository.Verify(x => x.Add(It.IsAny<PimsAcquisitionFileNote>()), Times.Once);
```

Comment:
In this instance I'd be tempted to check that the note that is going to be added is as expected.